internal/control_plane/command/root.go

```diff
@@ -18,34 +15,15 @@ var rootCmd = &cobra.Command{
 
 // Execute the root command and no error returned
 func Execute() {
+	rootCmd.AddCommand(start.NewCmd())
 	if err := rootCmd.Execute(); err != nil {
-		logger.Error("error in root command execution", err)
+		logger.Log.Err(err).Msg("Root command error")
+	} else {
+		logger.Log.Info().Msg("Root Command Executed")
```

Comment:
I guess this is redundant here. Can be deleted.

Reply:
Fixed it! , Line 21 and 22 were redundant, 20 is used when invalid command is entered